repo/mongodb/repo.go

```diff
@@ -34,9 +34,12 @@ import (
 	"github.com/looplab/eventhorizon/uuid"
 )
 
-// ErrModelNotSet is when an model factory is not set on the Repo.
+// ErrModelNotSet is when a model factory is not set on the Repo.
 var ErrModelNotSet = errors.New("model not set")
 
+// ErrNoCursor is when a provided callback function returns a nil cursor.
+var ErrNoCursor = errors.New("no cursor")
```

Comment:
Good that you added this back, however I did the same as part of the outbox branch which is about to merge soon. How do you feel about the error name I choose there? https://github.com/looplab/eventhorizon/pull/335/files#diff-101473a1d1cd877bc18bac496dde3677310222da489d26815d8b1e4bff9c655dR230

Reply:
Hey, I'm actually kind of fine with both error names, I can revert my changes in that regard if you want

Reply:
Lets go with your name then!